legend-engine-server/src/test/java/org/finos/legend/engine/server/test/shared/PureTestHelper.java

```diff
@@ -141,6 +141,12 @@ public static ServersState initEnvironment(boolean withH2, String serverConfigFi
             h2Server = AlloyH2Server.startServer(relationalDBPort);
             System.out.println("H2 database started on port:" + relationalDBPort);
         }
+        else
+        {
```

Comment:
why this change?

Reply:
so that we only set planLocal and exceplan properties for db tests , shouldnt affect other tests